clang/test/Driver/sycl-native-cpu.cpp

```diff
@@ -1,5 +1,5 @@
-// RUN: %clangxx -fsycl-device-only -fsycl-targets=native_cpu %s -### 2>&1 | FileCheck %s
-// RUN: %clangxx -fsycl-device-only -fsycl-targets=native_cpu --target=aarch64-unknown-linux-gnu %s -### 2>&1 | FileCheck %s --check-prefix=CHECK-AARCH64
+// RUN: %clangxx -fsycl-device-only -fsycl-targets=native_cpu %s -### -fno-sycl-libspirv 2>&1 | FileCheck %s
```

Comment:
NIT: can you keep the grouping of the options before `-###` and `%s`?

```suggestion
// RUN: %clangxx -fsycl-device-only -fsycl-targets=native_cpu -fno-sycl-libspirv %s -### 2>&1 | FileCheck %s
```